src/network/websockets/server/types.ts

```diff
@@ -16,6 +16,7 @@ export type KV = K & {
 export type Params = {
   onAuthenticate?: (params: {
     token: string;
+    close: () => void;
```

Comment:
I’m very keen on alphabetizing items where no other order is present. I’d put close above token. But also, feel free to ignore.